v3/README.md

```
@@ -1,9 +1,33 @@
+## What Beta means
+
+The v3 API in `v3/pkg/application` and related packages is stable. We will not make breaking changes without a deprecation cycle and clear migration guidance.
 
 ## Getting Started
 
-All the instructions for getting started are in the v3 documentation directory: `mkdocs-website`. 
-Please read the README.md file in that directory for more information.
+Full documentation is available at [v3.wails.io](https://v3.wails.io).
+
+To get started with a new project:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)